addons/map/XEH_postInitClient.sqf

```diff
@@ -8,52 +8,6 @@ LOG(MSG_INIT);
 // Calculate the maximum zoom allowed for this map
 call FUNC(determineZoom);
 
-[{
-    if (isNull findDisplay 12) exitWith {};
```

Comment:
I think we still want this line or the effects get added to the briefing maps (maps shown before mission starts)
e.g. night mission, before there were no lighting effects on map
I think to filter them out we need to do something like
```
[{
    params ["_display"];
    private _control = _display displayCtrl IDC_MAP;

    if (_display != findDisplay 12) exitWith {};
    ...
}, _this] call CBA_fnc_execNextFrame; // need to delay a frame or findDisplay doesn't work
````

Reply:
```sqf
if (ctrlIDD _display != IDC_MAP) exitWith {};
```
?

Reply:
little easier with that command :laughing:, `IDD_MAIN_MAP`

Reply:
Done.